L03-generic-collections/src/main/java/ru/chupaYchups/otus/DiyArrayList.java

```diff
@@ -0,0 +1,224 @@
+package ru.chupaYchups.otus;
+
+import java.util.*;
+
+/**
+ * Имплементация индексированного списка
+ * в которой реализована:
+ * - добавление элементов
+ * - перебор элементов
+ * - замена элементов
+ */
+public class DiyArrayList<T> implements List<T> {
+
+    public static Integer DEFAULT_CAPACITY = 10;
```

Comment:
Почему `Integer`, а не `int`?
```suggestion
    public static int DEFAULT_CAPACITY = 10;
```